components/Home/ContactInformation/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { FC } from 'react'
+import { ResponsiveLayoutGrid } from '../../Core'
+import styles from './ContactInformation.module.css'
+
+interface ContactInformationProps {
+  information: ContactInformationData[]
+}
+
+const ContactInformation: FC<ContactInformationProps> = ({ information }) => {
+  const renderInformation = information.map(info => (
```

Comment:
create separate function, render in JSX section instead of separate variable

Reply:
I'm not sure I understand, can you give some more details?

Reply:
`renderInformation` is a variable with Verb in it. Bit hard to comprehend. I would personally create "InfoDetail" component and just map things out in JSX below, instead of premapping them above JSX section

Reply:
Roger roger! I overall agree on splitting things down to smaller components